README.md

```diff
@@ -40,30 +40,28 @@ npm install redux-thunk
 yarn add redux-thunk
 ```
 
-The thunk middleware is the default export.
+The thunk middleware is a named export.
 
 <details>
 <summary><b>More Details: Importing the thunk middleware</b></summary>
 
 If you're using ES modules:
 
 ```js
-import thunk from 'redux-thunk' // no changes here 😀
+import { thunk } from 'redux-thunk'
 ```
 
-If you use Redux Thunk 2.x in a CommonJS environment,
-[don’t forget to add `.default` to your import](https://github.com/reduxjs/redux-thunk/releases/tag/v2.0.0):
+If you use Redux Thunk in a CommonJS environment:
 
-```diff
-- const thunk = require('redux-thunk')
-+ const thunk = require('redux-thunk').default
+```js
+const { thunk } = require('redux-thunk')
 ```
 
 Additionally, since 2.x, we also support a
 [UMD build](https://unpkg.com/redux-thunk/dist/redux-thunk.min.js) for use as a global script tag:
 
 ```js
-const ReduxThunk = window.ReduxThunk
+const ReduxThunk = window.ReduxThunk.thunk
 ```
```

Comment:
this can probably be removed altogether, I'm pretty sure we removed the UMD build in the major